Wdvh/absolutecard/absolutecard.py

```diff
@@ -0,0 +1,456 @@
+            *self.__series_count_text_black_stroke(),
+            f'-annotate +{width1+width2}+{height-25} "{self.episode_text}"',
+            *self.__series_count_text_effects(),
+            f'-annotate +{width1+width2}+{height-25} "{self.episode_text}"',
+            f'"PNG32:{self.__SERIES_COUNT_TEXT.resolve()}"',
+        ])
+
+        self.image_magick.run(command)
+
+        return self.__SERIES_COUNT_TEXT
+
+
+    def _combine_titled_image_series_count_text(self, titled_image: Path,
+                                                series_count_image: Path)->Path:
+        """
+        Combine the titled image (image+gradient+episode title) and the series
+        count image (optional season number+optional dot+episode number) into a
+        single image. This is written into the output image for this object.
+
+        :param      titled_image:       Path to the titled image to add.
+        :param      series_count_image: Path to the series count transparent
+                                        image to add.
+
+        :returns:   Path to the created image (the output file).
+        """
+
+        command = ' '.join([
+            f'composite',
+            f'-gravity center',
+            f'-geometry +0+790.2',
+            f'"{series_count_image.resolve()}"',
+            f'"{titled_image.resolve()}"',
+            f'"{self.output_file.resolve()}"',
+        ])
+
+        self.image_magick.run(command)
+
+        return self.output_file
+
+
+    @staticmethod
+    def is_custom_font(font: 'Font') -> bool:
+        """
+        Determines whether the given font characteristics constitute a default
+        or custom font.
+        
+        :param      font:   The Font being evaluated.
+        
+        :returns:   True if a custom font is indicated, False otherwise.
+        """
+
+        return ((font.file != StandardTitleCard.TITLE_FONT)
+            or (font.size != 1.0)
+            or (font.color != StandardTitleCard.TITLE_COLOR)
+            or (font.replacements != StandardTitleCard.FONT_REPLACEMENTS)
+            or (font.vertical_shift != 0)
+            or (font.interline_spacing != 0)
+            or (font.kerning != 1.0)
+            or (font.stroke_width != 1.0))
+
+
+    @staticmethod
+    def is_custom_season_titles(custom_episode_map: bool, 
+                                episode_text_format: str) -> bool:
+        """
+        Determines whether the given attributes constitute custom or generic
+        season titles.
+        
+        :param      custom_episode_map:     Whether the EpisodeMap was
+                                            customized.
+        :param      episode_text_format:    The episode text format in use.
+        
+        :returns:   True if custom season titles are indicated, False otherwise.
+        """
+
+        standard_etf = StandardTitleCard.EPISODE_TEXT_FORMAT.upper()
+
+        return (custom_episode_map or
+                episode_text_format.upper() != standard_etf)
```

Comment:
```suggestion
        return False
```
Your card doesn't use any season titles